src/include/sof/trace/preproc.h

```diff
@@ -158,8 +158,10 @@
 		return_t META_CONCAT(prefix, postfix) (args)
 
 /* counteract compiler warning about unused variables */
+#ifndef UNUSED
 #define UNUSED(arg1, ...) do { META_RECURSE( \
 	META_MAP_AGGREGATE(1, _META_VOID2, _META_VOID(arg1), __VA_ARGS__)); \
 	} while (0)
+#endif
```

Comment:
I feel this change is not good. Would it be better to rename UNUSED() to SOF_UNUSED() or SOF_TRACE_UNUSED()?

Reply:
Yes please rename this to SOF_TRACE_UNUSED() in a separate commit (or even separate PR).

> I Think the pre-processing code is now only used for logging, could be deprecated...

... for Intel.